support/faq.md

```diff
@@ -2,60 +2,223 @@
+it will re-trigger a webhook challenge request,and eventually a webhook re-activation.
+
+While your webhook is marked Disabled, Adobe will continue to log events a Journal, 
+you can retrieve all of your events for the past 7 days using [Journaling](../intro/journaling_intro.md).
+
+#### Does every Adobe I/O Events webhook http requests come with a signature? 
+     
+Yes, to allow your webhook to reject forged requests, 
+Adobe I/O Events adds a  [`x-adobe-signature`](../intro/webhooks_intro.md#authenticating-events) 
+header to every single Http request it does to your webhook URL (even the first `challenge` GET request)
+      
+#### Do Adobe I/O Events notifications come from a range of static IPs? 
+
+We had a few customers asking this in the context of securing their endpoint; 
+their requirement: accepting traffic only from a range of static IPs.
+
+The answer is no. Adobe I/O Events notifications services are hosted on AWS and Azure,
+their IPs change over time.
+
+Reminder: each Adobe I/O Events http request
+holds an signature header (see the previous question), however if this is not enough 
+and if the above is a non-negotiable requirement, 
+you may choose to use the pull model instead, 
+and leverage our [Journaling API](../intro/journaling_intro.md).          
+
+
+#### What is the size of notifications when in batch delivery style?
+     
+When registering a webhook to receive Adobe I/O Events notifications, you can select the delivery style:
+* either receiving one event at a time ("Single"): each event resulting in an HTTP request to the webhook URL
+* or multiple events together ("Batch"): in this case, HTTP requests will still remain near-real time,  
+the batch size will vary depending on the incoming traffic 
+and the batch size will be at max 2MB bytes and will contain a maximum of 100 events.
+
+
+## Journaling FAQ
+
+#### How far back are I/O Events available via the Journaling API?
+
+Adobe I/O stores 7 days of subscribed events that can be retrieved via the Journaling API.
+
+#### Why do I only get one event, irrespective of the `limit` I use?
+
+Our Journaling API `limit` parameter is used to specify the “maximum” number of events 
+that may be returned by the API.
+It is used to specify an upper bound to the number of events to ensure that the consumer 
+can handle the response gracefully. The behavior you observe is expected. 
+It is perfectly ok to get 1 event when you specify a limit greater than 1 . 
+The number of events that gets returned in a response 
+can vary depending on the incoming traffic (upper bound being equal to the limit parameter).
+See our [Journaling API documentation](../api/journaling_api.md#limiting-the-size-of-the-batch) for more details.
+     
+#### Is there a way to get the list of events all together at once?
+     
+No, this query is not supported in our journaling API,
+however, using the `since` parameter you can to follow the journal
```

Comment:
you can (extraneous 'to') follow the journal

Reply:
done